src/main/java/com/ecore/roles/web/rest/TeamsRestController.java

```diff
@@ -24,7 +21,7 @@ public class TeamsRestController implements TeamsApi {
     private final TeamsService teamsService;
 
     @Override
-    @PostMapping(
+    @GetMapping(
```

Comment:
In order to follow the Rest specification, of course, if the there isn’t any other security reason or URL length limitation